langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -99,6 +119,22 @@ public isolated function get(xml<ItemType> x, int i) returns ItemType = @java:Me
 } external;
 
 # Concatenates xml and string values.
+# 
+# ```ballerina
+# xml bookA = xml `<book>Ballerina</book>`;
+# xml bookB = xml `<book>Java</book>`;
+# xml bookC = xml `<book>Python</book>`;
+# xml:concat(bookA, bookB, bookC) ⇒ <book>Ballerina</book><book>Java</book><book>Python</book>
+# 
+# bookA.concat(bookB) ⇒ <book>Ballerina</book><book>Java</book>
+# 
+# bookC.concat("Coding") ⇒ <book>Python</book>Coding
+# 
+# xml:concat("Hello", "World") ⇒ HelloWorld
+# 
+# xml[] subjectList = [xml `<subject>English</subject>`, xml `<subject>Math</subject>`, xml `<subject>ICT</subject>`];
+# xml:concat(...subjectList) ⇒ <subject>English</subject><subject>Math</subject><subject>ICT</subject>
```

Comment:
```suggestion
# xml[] subjects = [xml `<subject>English</subject>`, xml `<subject>Math</subject>`, xml `<subject>ICT</subject>`];
# xml:concat(...subjects) ⇒ <subject>English</subject><subject>Math</subject><subject>ICT</subject>
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18